beater/route_config.go

```diff
@@ -35,6 +37,8 @@ import (
 var (
 	rootURL = "/"
 
+	agentConfigURL = "/config"
```

Comment:
Have you considered nesting under an agent namespace, e.g. `/agent/config`, and also have you thought about versioning? 
In case we are introducing a `config` endpoint to fetch the servers config, would that fit into `/config`?

Reply:
1. - I didn't thing about the endpoint name at all. I can change it to anything. @graphaelli are you good with the suggestion? where should the version go? `/agent/v1/config`?
2. - don't understand the question, this endpoint is to expose config, not fetching it - why would I need an endpoint to expose the server config?

Reply:
Fetching the current APM Server config might be interesting for the environment in which the server is running (cloud, k8s, ..), but I mainly used it to prove my point. When I have a server exposing an endpoint `config` I would expect it to return the servers config at least if nothing else is requested via query params.

Reply:
I suggest `/v1/agent/configs`, the `agent` being the namespace under which the `config` resource is nested, and aligned with how we introduced versioning for the intake API.